modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js

```diff
@@ -243,6 +243,35 @@ function convertNodes(
   }
 }
 
+/**
+ * Generate transformation matrix for node
+ * @param node
+ * @param matrix
+ */
+function getCompositeTransformationMatrix(node, matrix) {
```

Comment:
Types please?

Reply:
Will do in separate PR where we migrate this file from `js` -> `ts`